invenio_app_rdm/theme/views.py

```diff
@@ -32,6 +32,7 @@ def create_blueprint(app):
     blueprint.add_url_rule(routes["index"], view_func=index)
     blueprint.add_url_rule(routes["robots"], view_func=robots)
     blueprint.add_url_rule(routes["help_search"], view_func=help_search)
+    blueprint.add_url_rule(routes["help_statistics"], view_func=help_statistics)
```

Comment:
shelve, for general discussion: I am not sure if having `APP_RDM_ROUTES` config variable to configure urls is the best